docs/howto/windows.txt

```diff
@@ -16,52 +19,64 @@ Install Python
+the latest Python version. Download the EXE file and run it.
 
-After installation, open the command prompt and check the Python version by
-executing ``python --version``. If you encounter a problem, make sure you
-have set the ``PATH`` variable correctly. You might need to adjust your
-``PATH`` environment variable to include paths to the Python executable and
-additional scripts. For example, if your Python is installed in
-``C:\Python34\``, the following paths need to be added to ``PATH``::
+We recommend checking ``Add Python 3.5 to PATH``, and then clicking ``Install
+Now`` to include IDLE, pip and documentation.
 
-    C:\Python34\;C:\Python34\Scripts;
+After installation, open the command prompt and check the Python version 
+matches the version you installed by executing ``python --version``.
 
-Install Setuptools
-==================
+About pip
+=========
 
-To install Python packages on your computer, Setuptools is needed. Download the
-latest version of `Setuptools <https://pypi.python.org/pypi/setuptools>`_ for
-your Python version and follow the installation instructions given there.
+``pip`` is a package manage for Python. It makes installing and uninstall Python
+packages (such as Django!) very easy. For the rest of the installation, we will
+use ``pip`` to install Python packages from the command line.
 
-Install PIP
-===========
+Install virtualenv and virtualenvwrapper (optional)
+===================================================
 
-`PIP <https://pip.pypa.io/>`_ is a package manager for Python that
-uses the `Python Package Index <https://pypi.python.org>`_ to install Python
-packages. PIP will later be used to install Django from PyPI. If you've
-installed Python 3.4, ``pip`` is included so you may skip this section.
+``virtualenv`` and ``virtualenvwrapper`` provide a dedicated environment for each
+Django Project you create. While not mandatory, this is considered a best
+practice and will save you time in the future when you're ready to deploy your
+project. Simply type::
 
-Open a command prompt and execute ``easy_install pip``. This will install
-``pip`` on your system. This command will work if you have successfully
-installed Setuptools.
+    pip install virtualenvwrapper-win
 
-Alternatively, go to `<https://pip.pypa.io/en/latest/installing.html>`_
-for installing/upgrading instructions.
+Then create a virtualenv for your project::
+
+    mkvirtualenv myproject
 
 Install Django
 ==============
 
 Django can be installed easily using ``pip``.
 
-In the command prompt, execute the following command: ``pip install django``.
-This will download and install Django.
+In the command prompt, execute the following command::
+
+    pip install django
+
+This will download and install the latest Django release.
 
 After the installation has completed, you can verify your Django installation
 by executing ``django-admin --version`` in the command prompt.
 
+Testing in a Browser
```

Comment:
In my opinion, to limit the amount of documentation we maintain we should keep this document limited to things that are Windows specific. This section doesn't seem to meet that criteria.